openedx/core/djangoapps/embargo/tests/factories.py

```diff
@@ -1,3 +1,6 @@
+"""Factories for generating fake embargo data."""
+# pylint: disable=missing-docstring
```

Comment:
Looks like you added a docstring, so we should be able to remove this pylint suppresion :)